src/parser/vast_parser.js

```diff
@@ -27,6 +27,7 @@ export class VASTParser extends EventEmitter {
   constructor() {
     super();
 
+    this.parseDuration = parserUtils.parseDuration;
```

Comment:
One discutions we had about exposing/exporting util functions is that we can either do like you did but then we need to create a new instance of vastParser otherwise the second solution would be to export utils function as a standalone then we could use it without creating a new instance and in addition if we don't use it we could benefit from tree-shaking